onnxruntime/core/framework/execution_frame.h

```diff
@@ -66,7 +71,7 @@ class IExecutionFrame {
    * write the output values to the 'fetches' vector
    * Don't access the values after SessionState is destroyed 
```

Comment:
>Don't access the values after SessionState is destroyed  [](start = 5, length = 56)

this is no longer true when release, right?

Reply:
updated the comment.

---
In reply to: [598070981](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598070981) [](ancestors = 598070981)